samples/schema/petstore/mysql/mysql_schema.sql

```diff
@@ -95,6 +95,7 @@ CREATE TABLE IF NOT EXISTS `Capitalization` (
 --
 
 CREATE TABLE IF NOT EXISTS `Cat` (
+  `declawed` TINYINT(1) DEFAULT NULL,
```

Comment:
Looks like `{{{#vars}}}{{{/vars}}}` array now has duplicated values. It breaks current implementation.